docs/dashboards/dashboards-overview.asciidoc

```diff
@@ -6,12 +6,19 @@
 :frontmatter-tags-content-type: [overview]
 :frontmatter-tags-user-goals: [visualize, monitor, analyze]
 
-The following sections describe the {security-app}'s prebuilt dashboards, which provide visualizations of your security environment.
+The {security-app}'s default dashboards provide useful visualizations of your security environment. To view them in {security-app}, select *Dashboards* from the navigation menu.
+From the Dashboards page, you can access the default dashboards, as well as create and access custom dashboards.
 
-You can also create and access custom security dashboards from the Dashboards landing page. To create one, click **Create Dashboard**. Custom dashboards must have the tag `Security Solution` to appear on the Dashboards page.
+To create a new custom dashboard, click **Create Dashboard**. You can control which custom dashboards appear in the table by:
+
+* Using the text search field to filter by name or description
+* Using the **Tags** menu to filter by tag
+* Clicking one of a custom dashboard's tags to toggle whether to filter for that tag
+To create a new tag or edit existing tags, open the **Tags** menu and click **Manage tags**.
```

Comment:
A few fixes here. Partly subjective: basic imperative verbs (vs. gerunds) are more direct, especially for commands/suggestions, and they tend to encourage slower reading through the whole list (vs. quicker scanning). Also, need another blank line to end the list and start a new paragraph.
```suggestion
To create a new custom dashboard, click **Create Dashboard**. You can control which custom dashboards appear in the table:

* Use the text search field to filter by name or description.
* Use the **Tags** menu to filter by tag.
* Click a custom dashboard's tags to toggle filtering for each tag.

To create a new tag or edit existing tags, open the **Tags** menu and click **Manage tags**.
```